src/Illuminate/Support/ServiceProvider.php

```diff
@@ -507,7 +507,7 @@ public static function addProviderToBootstrapFile(string $provider, string $path
 '.$providers.'
 ];';
 
-        file_put_contents($path, $content);
+        file_put_contents($path, $content.PHP_EOL);
```

Comment:
This is maybe the wrong EOL character. Even on Windows, it's usual practice to still use `\n`.

Reply:
We are using the PHP_EOL on the line 502. So I guess, if we were to apply the fix your suggesting, needs to be on that place too.

Reply:
Yeh, I'd say so.